### DISCUSSION TITLE
Support Subscript and superscript in the WorkBench

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5828] [Feature Requests]
**In the WorkBench:**
<img width="405" alt="image" src="https://github.com/specify/specify7/assets/37256050/f47ae0b6-6167-4e4d-9b79-f8c7e7baf257">

**In Excel:**
<img width="430" alt="image" src="https://github.com/specify/specify7/assets/37256050/7cddc142-2c37-4c00-afee-b9d549ad0c86">

[Subscript and superscript](https://en.wikipedia.org/wiki/Subscript_and_superscript)

This is important for importing chemical composition of minerals

**Comment by maxpatiiuk** [comment]
Could you if any of the characters on this page work? https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts
